packages/data/src/atomic-store/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { mapValues } from 'lodash';
+
+export function createAtomicStore( config, registry ) {
+	// I'm probably missing the atom resolver here
+	const selectors = mapValues( config.selectors, ( atomSelector ) => {
+		return ( ...args ) => {
+			return atomSelector(
+				registry.__unstableGetAtomResolver()
```

Comment:
You probably meant to check its existence here and not call it, no?

Reply:
no, this is correct actually, it's a getter for the resolver but it doesn't call the resolver.